examples/_test-amino-bridge-withdraw.ts

```diff
@@ -0,0 +1,64 @@
+import * as BIP39 from "bip39";
+import { MsgWithdrawToken } from "../lib/codec/Switcheo/carbon/bridge/tx";
+import { CarbonSDK, CarbonTx, CoinModule } from "./_sdk";
+import "./_setup";
+import Long from "long";
+import BigNumber from "bignumber.js";
+import { Duration } from "../lib/codec";
+
+(async () => {
+  const mnemonics = process.env.MNEMONICS ?? BIP39.generateMnemonic();
+  console.log("mnemonics", mnemonics);
+
+  const sdk = await CarbonSDK.instanceWithMnemonic(mnemonics, { network: CarbonSDK.Network.LocalHost });
+  console.log('sdk initialized')
+
+  const externalTokensResult = await sdk.query.bridge.ExternalTokenAll({ bridgeId: new Long(0), chainId: '', denom: '' })
+
+  const externalToken = externalTokensResult.externalTokens[0]
+  const externalTokenCarbonDenom = externalToken?.denom ?? ''
+
+  const body = JSON.stringify({
+    address: sdk?.wallet?.bech32Address,
+    coins: ['1000000000swth'],
+  })
+  await fetch(sdk.networkConfig.faucetUrl, { method: 'POST', body })
+  console.log('Minted SWTH for gas')
+
+  const tokenDp = sdk.token.getDecimals(externalTokenCarbonDenom) ?? 0
+  const params: CoinModule.MintTokenParams = {
+    amount: new BigNumber(10000).shiftedBy(tokenDp),
+    denom: externalTokenCarbonDenom,
+  }
+  const mintBrdgToken = await sdk.coin.mintToken(params)
+  console.log('Minted brdg tokens', mintBrdgToken)
+
+  const balancesResult = await sdk.query.coin.Balances({ address: sdk.wallet.bech32Address })
+  console.log('balances:', balancesResult)
+
+  const expiryDuration = Duration.fromPartial({
+    seconds: new Long(1000),
+  })
+
+  const result = await sdk.wallet.sendTxs([{
+    typeUrl: CarbonTx.Types.MsgWithdrawToken,
+    value: MsgWithdrawToken.fromPartial({
+      creator: sdk.wallet.bech32Address,
+      connectionId: externalToken?.connectionId ?? '',
+      receiver: sdk.wallet.evmHexAddress,
+      tokens: {
+        denom: externalToken?.denom ?? '',
+        amount: new BigNumber(10).shiftedBy(tokenDp).toString(10),
+      },
+      relayFee: {
+        denom: 'swth',
+        amount: '100',
+      },
```

Comment:
Think it's a good practice to use `Coin.fromPartial({ ... })` to ensure the type is correct.